objects/cancel_invoice_output.go

```diff
@@ -1,8 +1,6 @@
 // Copyright ©, 2023-present, Lightspark Group, Inc. - All Rights Reserved
 package objects
 
-import "github.com/lightsparkdev/go-sdk/types"
```

Comment:
this should not be removed

Reply:
yeahhhh i figured 😢 maybe im using the generator wrong